test-framework/junit5/src/main/java/io/quarkus/test/junit/NativeImageTest.java

```diff
@@ -24,9 +24,12 @@
  * in native image tests. Such injection is only possible in tests injected with
  * {@link @QuarkusTest} so the test class structure must take this into account.
  *
+ * @deprecated
+ *             Use {@link @QuarkusIntegrationTest} instead.
```

Comment:
Any reason this isn't on the same line as `@deprecated`?

Reply:
Hey @geoand the `@Deprecated` annotation on that has the `since` and `forRemoval` attributes. I didn't put anything in the `since` attribute because I didn't know what Quarkus version this would be part of.

Should I put something in `since` and set `forRemoval=true`?

Reply:
Yes please

Reply:
What should I put in the `since` field? `2.8.0.Final`?